node/tessel-export.js

```diff
@@ -952,6 +957,303 @@ Tessel.LED.prototype.read = function(callback) {
   });
 };
 
+Tessel.Wifi = function() {
+  var state = {
+    settings: {},
+    connected: false
+  };
+
+  Object.defineProperties(this, {
+    isConnected: {
+      get: () => state.connected
+    },
+    connected: {
+      set: (value) => {
+        state.connected = value;
+      }
+    },
+    settings: {
+      get: () => state.settings,
```

Comment:
No, I wouldn't recommend that. I didn't have a chance to do experiments today, but will hit T this weekend and report back, then we figure out a way forward